object_tracker/match.py

```diff
@@ -249,7 +254,8 @@ def __call__(self, image: ndarray) -> List[int]:
         # check if there are enough features for using optical flow
         # if not and use_orb flag is set, the method tries to find
         # features using orb detector
```

Comment:
It's not return score, how does it fit to the structure that the tracker return?

Reply:
You are correct one of the future works is to add score for OF but for now it does not exist.